examples/hybrid-compute/README.md

```diff
@@ -0,0 +1,85 @@
+# Getting Started with Quantinuum's Hybrid Compute Feature
+
+Quantinuum's Hybrid Compute feature enables users to run classical operations within qubit coherence times. This enables research in quantum error correction and new algorithm designs not otherwise available. This guide covers how to set up and use this feature.
+
+Hybrid Compute is enabled via the use of [Web Assembly (WASM)](https://webassembly.org/), which enables fast, real-time computations.
+
+Wasm itself is a binary instruction format. To write in and compile programs to it, another language such as Rust or C++ is used to write your desired classical functions and compile these to Wasm. The quantum computing workflow will then contain a call to Wasm. This folder contains examples of this workflow. Wasm is used because it is a fast, safe, expressive, and easy to compile to using familiar programming languages. 
+
+## Organization
+
+This subfolder is organized as follows.
+
+---
+    ├── Hybrid-compute        <- Examples of how to use Hybrid Compute via WASM
+          |
+          ├── RUS_WASM        <- Repeat Until Success source code
+---
+
+## Download
+
+If you haven't done so already, download the complete set of examples on the user portal by clicking the **Download** button on the bottom-left of the folder viewer.
+
+## Notebook Example
+
+The python notebook in this directory called `repeat_until_success_with_wasm.ipynb` can be run to show an example of the hybrid compute feature's workflow, without needing to edit any classical functions or compile to Wasm.
+
+## Setup
+
+To create and use your own Hybrid Compute functions in your workflow, it is necessary to use Wasm and a language that compiles code to Wasm. These instructions contain the steps for setting up an environment to do this.
+
+It is recommended to either use Rust or C++ for the compilation language to Wasm. The languages have functionality that easily compiles to Wasm. For other language options see [Wasm Getting Started Developer's page](https://webassembly.org/getting-started/developers-guide/).
+
+### Option 1: Rust
+
+If you're new to Rust, see the [The Rust Programming Language](https://doc.rust-lang.org/book/title-page.html) book. Note that Cargo is the Rust package manager. For more info on Cargo, see [The Cargo Book](https://doc.rust-lang.org/cargo/guide/).
+
+If using Visual Studio code, you can also install the `rust-analyzer` extension in the Extensions tab.
+
+#### Install Rust
+
+Install Rust using the [Rust Installer](https://www.rust-lang.org/tools/install).
+
+**Windows Users:** Before running the installer, you'll need to install the [Microsoft C++ Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/).
```

Comment:
This is also relevant on linux, but most people will have some kind of compiler already installed? Not sure about mac.